app/invoices/employer_invoice.rb

```diff
@@ -24,7 +24,7 @@ def save
         FileUtils.mkdir_p(invoice_folder_path)
       end
       pdf_doc.render_file(invoice_absolute_file_path) unless File.exist?(invoice_absolute_file_path)
-      join_pdfs [Rails.root.join('tmp', invoice_absolute_file_path), Rails.root.join('lib/pdf_templates', 'ma_non_discrimination_and_language_tags.pdf')]
+      join_pdfs [Rails.root.join('tmp', invoice_absolute_file_path), Rails.root.join('lib/pdf_templates', '#{fetch_invoices_addendum}')]
```

Comment:
You going to have to condition this and not perform `join_pdfs` when there is no fetch_invoices_addendum value.